### PR TITLE
Skipping test since Mirror with policer is not supported on cisco-8000 platform

### PR DESCRIPTION
**Pull request by vperumal**
…forms
### Description of PR
Skipping test since mirror with policer is not supported on Cisco 8000 platforms.

Summary:
Fixes # (issue)

### Type of change
- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
Skipping test since mirror with policer is not supported on Cisco 8000 platforms.

### CODE REVIEW

**Review by yxieca — APPROVED** [review]

**Comment by sachinv-msft** [comment]
@yxieca could you please merge this PR?

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -15,6 +15,12 @@ platform_tests/fwutil/test_fwutil.py::test_fwutil_auto:
   skip:
     reason: "Command not yet merged into sonic-utilites"
 
+everflow/test_everflow_testbed.py::TestEverflowV4IngressAclIngressMirror::test_everflow_dscp_with_policer:
+  skip:
```
Comment (yxieca) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:19]: Can we do xfail with strict being true instead? This way we know there is a feature gap.

Comment (vperumal) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml]: @yxieca : Made the changes as per your comments

**Review by wangxin — APPROVED** [review]

**Comment by wangxin** [comment]
@vperumal Can you resolve the merge conflicts?

**Comment by vperumal** [comment]
Hi @wangxin,

I have resolved the conflict. Kindly take a look

**Comment by sachinv-msft** [comment]
@wangxin could you please merge the PR?

**Pull request merged**